webapp/transport/models.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models
+from django.utils.text import slugify
+from django_extensions.db.models import TimeStampedModel
+from django.conf import settings
+from ordering.models import Supplier, OrderRound
+
+
+class Route(TimeStampedModel):
+    class Meta:
+        verbose_name = 'Route'
+        verbose_name_plural = 'Routes'
+
+    name = models.CharField(max_length=100)
+    suppliers = models.ManyToManyField(Supplier)
+    slug = models.SlugField(unique=True, editable=False, max_length=100)
+
+    def save(self, **kwargs):
+        self.slug = slugify(self.name)
+        return super(Route, self).save(**kwargs)
+
+    def __str__(self):
+        return self.name
+
+    @property
+    def suppliers_names(self):
+        return list(map(lambda s: s.name, self.suppliers.all()))
+
+
+class Ride(TimeStampedModel):
+    class Meta:
+        verbose_name = 'Ride'
+        verbose_name_plural = 'Rides'
+
+    order_round = models.ForeignKey(
+        OrderRound, models.SET_NULL, null=True, related_name="ride")
+    route = models.ForeignKey(
+        Route, models.SET_NULL, null=True, related_name="rides")
+    driver = models.ForeignKey(
+        settings.AUTH_USER_MODEL, related_name="rides_as_driver")
+    codriver = models.ForeignKey(
+        settings.AUTH_USER_MODEL, related_name="rides_as_codriver")
+    coordinators = models.ManyToManyField(
```

Comment:
Wait, are these the Transport or "Uitdeel" (distribute?) coordinators?

Reply:
Transport coordinators. So that driver and codriver have someone to reach in case of issues.

Reply:
Ow wait you think I should make that explicit in the name?

Reply:
> Ow wait you think I should make that explicit in the name?
> But why would one add a "uitdeel" coordinator to a Ride?

Nope, I just asked because I'm not super familiar with the process.